cmds/checkup.go

```diff
@@ -124,19 +128,58 @@ func (cu *Checkup) inspectFile(file string) (translatedStrings []string, err err
 
 	ast.Inspect(astFile, func(n ast.Node) bool {
 		switch x := n.(type) {
+		case *ast.AssignStmt:
+			// use a hashmap for defined variables to a list of reassigned variables sharing the same var name
```

Comment:
I wonder if and how useful it would be to have helper private functions for each of the case statements. WDYT?

Reply:
I like that idea. Make it easier to read the parsing. Let me make the modifications.